src/os/ui/slick/slickgrid.js

```diff
@@ -247,6 +245,18 @@ os.ui.slick.SlickGridCtrl = function($scope, $element, $compile) {
     this.dataView.syncGridSelection(this.grid, false);
   }
 
+  if (options['editable']) {
+    // editable grids have additional CSS styles. add the class to enable them.
+    this.element.addClass('editable');
```

Comment:
Is this a slick-grid specific class? We usually try to be more specific with css class names when deviating from using a bootstrap style. For instance `c-slickgrid__editable` indicates that the class is for the slick grid component. It helps to be specific since `editable` could apply to a number of things

Reply:
This logic was already in the file before my changes; I only moved it down a few lines rather than create a second `if (options['editable'])` block. Changing class names will have a bigger impact elsewhere and is not part of my scope.

Reply:
Good point. Thanks for clarifying!